README.md

```diff
@@ -100,16 +103,37 @@ cd /srv/atd-airflow;
 # pull the changes
 git pull;
 
-# return to user-land
-exit;
+
 ```
 
+#### If there has been a change to the Airflow Docker image like when updating the Airflow verson:
+
 The production Airflow deployment uses a second Docker compose file which provides haproxy configuration overrides. To start the production docker compose stack use you must load both files in order:
 
 ```shell
+# dts-int-data-p01
+
+# become the superuser
+su -;
+
+# enter into the production airflow directory
+cd /srv/atd-airflow;
+
+# pull the fresh production image
+docker pull atddocker/atd-airflow:production
```

Comment:
There couldn't be a less of a nothing request, but will you stick semicolons on lines of bash commands that are missing them? It'll do the right thing based on new lines, but should someone suck all the commands up into one line, which isn't uncommon, those semicolons break up the commands.

Reply:
yes! i will update to match the rest of the commands 🙏